server/routes/users.js

```diff
@@ -0,0 +1,60 @@
+const express = require('express');
+const { db, User, Order, Item, Sessions } = require('../db');
+const userRoute = express.Router();
+
+userRoute.get('/', async(req, res, next) => {
+    try {
+        res.send(await User.findAll())
```

Comment:
**This route fully exposes all users emails, passwords, etc to any user. This is a major privacy/security concern. We need to be much more careful and diligent with how we expose user data.**